ethapi/backend.go

```diff
@@ -57,7 +57,6 @@ type Backend interface {
 	// General Ethereum API
 	Progress() PeerProgress
 	SuggestGasTipCap(ctx context.Context, certainty uint64) *big.Int
-	EffectiveMinGasPrice(ctx context.Context) *big.Int
```

Comment:
Can you please also regenerate Backend mock after this change?

Reply:
Done, thanks!